hack/ics20-token-transfer.sh

```
@@ -0,0 +1,166 @@
+  exit 1
+fi
+
+# Check balances after transfer to Rollkit
+log_info "Checking balances after transfer to Rollkit..."
+log_info "Gaia user balance:"
+$GAIAD_BIN q bank balances $USER_ADDRESS_GAIA --node $GAIA_RPC --home "$CURRENT_DIR/testnet"
+
+log_info "Rollkit user balance (should show IBC tokens):"
+ibc_tokens_visible=false
+
+for i in {1..${20}}; do
```

Shellcheck (0.10.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)